kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,98 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+# Install litmus and wait until pod is running
+def install_litmus(version):
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-%s.yaml" % version)
+
+    runcommand.invoke("oc wait deploy -n litmus chaos-operator-ce --for=condition=Available --timeout=30s")
+
+
+def perform_experiment(version_string):
```

Comment:
More than perform, I'd say deploy or install since you're adding the generic experiment CRs, but there is no action taken that could be interpreted as an execution of the experiment.

Reply:
Update to "deploy_all_experiments". Thoughts here?